lib/resources/r4/family_member_history.yaml

```diff
@@ -1,28 +1,553 @@
+          "valueBoolean": true
+        }
+      ]
+    }
+
+- name: status
+  required: 'Yes'
+  type: code
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.status
+  description: A code specifying the status of the record of the family history of a specific family member.
+  action:
+    - create
+    - update
+    - terminology
+  example: |
+    {
+      "status": "partial"
+    }
+  binding:
+    description: A code that identifies the status of the family history record.
+    terminology:
+      - display: FamilyHistoryStatus
+        system: http://hl7.org/fhir/history-status
+        info_link: https://hl7.org/fhir/valueset-history-status.html
+
+
+- name: dataAbsentReason
+  required: 'No'
+  type: CodeableConcept
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.dataAbsentReason
+  description: Describes why the family member's history is not available.
+  action:
+    - create
+    - update
+    - terminology
+  example: |
+    {
+      "dataAbsentReason": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/history-absent-reason",
+            "code": "subject-unknown"
+          }
+        ]
+      }
+    }
+  binding:
+    description: Codes describing the reason why a family member's history is not available.
+    note: Can be subject-unknown or unable-to-obtain
+    terminology:
+      - display: FamilyHistoryAbsentReason
+        system: http://terminology.hl7.org/CodeSystem/history-absent-reason
+        info_link: https://hl7.org/fhir/valueset-history-absent-reason.html
+
+- name: patient
+  required: 'Yes'
+  type: Reference
+  description: The person who this history concerns.
+  action:
+    - create
+    - update
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/631923",
+        "display": "Nilsson, Christine"
+      }
+    }
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.patient
+
+- name: name
+  required: 'No'
+  type: string
+  description: The family member's name.
+  action:
+    - create
+    - update
+  example: |
+    {
+      "name": "Halpert, Simothy"
+    }
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.name
+
+- name: relationship
+  required: 'Yes'
+  type: CodeableConcept
+  description: The type of relationship this person has to the patient (father, mother, brother etc.).
+  url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.relationship
+  action:
+    - create
+    - update
+    - terminology
+  example: |
+    {
+      "relationship": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+            "code": "BRO"
+          }
+        ]
+      }
+    }
+  note: |
+    <ul>
+      <li>Should we explain what we are using 'FAMMEMB' code for?</li>
```

Comment:
yea... maybe something like:

a relationship of FAMMEMB indicates a FamilyMemberHistory that contains information specific to the patient that pertains to all relations of the patient.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L146